pkg/image/api/v1/types.go

```diff
@@ -201,19 +213,27 @@ type ImageStreamImage struct {
 
 // DockerImageReference points to a Docker image.
 type DockerImageReference struct {
```

Comment:
Does this object need `json` struct tags? Do not ever serialize it? if not, what is it doing in this file?